README.rst

```diff
@@ -0,0 +1,484 @@
+############################################
+                  KWIVER
+############################################
+
+Kitware Image and Video Exploitation and Retrieval
+
+
+The KWIVER toolkit is a collection of software tools designed to
+tackle challenging image and video analysis problems and other related
+challenges. Recently started by Kitware’s Computer Vision and
+Scientific Visualization teams, KWIVER is an ongoing effort to
+transition technology developed over multiple years to the open source
+domain to further research, collaboration, and product development.
+
+The project is structured with the parent `kwiver` repository working
+as CMake "super-build" that pulls in a number of KWIVER and other open
+source projects.
+
+
+Building KWIVER
+===============
+
+Fletch
+------
+
+KWIVER has (and will have more) a number of dependencies on 3rd party
+Open Source libraries.  To make it easier to build KWIVER, especially
+on systems like Microsoft Windows that don't have package manager,
+[Fletch](https://github.com/Kitware/fletch) was developed to gather,
```

Comment:
Need to change from markdown style hyperlinks to restructured text throughout this document